app/views/answers/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if @answer.errors.any? %>
+  <ul>
+    <% @answer.errors.full_messages.each do |msg| %>
+       <li class="error"><%= msg %></li>
+    <% end %>
+  </ul>
+<% end %>
+
+<%= form_for [@answer.question, @answer], url: question_answer_path(@answer.question, @answer), method: :patch do |f| %>
```

Comment:
If this form_for is correct (i.e. the @answer is already saved) rails should infer the right path and method.